docs/en/latest/deployment-modes.md

```diff
@@ -0,0 +1,157 @@
+---
+title: Deployment modes
```

Comment:
The `Deployment modes` work for me. Another option is `Deployment Options.` To make sure we're aligned with the community, @spacewander may I know which one you're recommended? 

1. Deployment Role
2. Deployment Modes
3. Deployment Options

Reply:
The document had both deployment mode and role in it. I made it just deployment mode. Deployment mode sounds good to me and conveys what it does effectively. If you have other ideas, I will change it.

Reply:
IMHO, `Deployment Modes` would be better